[Mhv 33641] Messages FAQ (unauth + auth) component

vsaleem opened the pull request:
## Description
Added new MessageFAQ component with container.  This is a static implementation for now but further development to add functionality and content to FAQs is planned for the future.

Unit tests for this component will be added in Sprint 89.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000
https://vajira.max.gov/browse/MHV-33641
https://vajira.max.gov/browse/MHV-33638


## Testing done


## Screenshots
**Authenticated User Desktop/Mobile View:**
![Screenshot (170)](https://user-images.githubusercontent.com/59583325/195386279-d020ebb6-69a4-4282-ae9d-9588585e2154.png)

![Screenshot (173)](https://user-images.githubusercontent.com/59583325/195386456-cc21fdd2-f5c5-4639-8841-02927b9fe321.png)

**Unauthenticated User Desktop/Mobile Views:**
![Screenshot (171)](https://user-images.githubusercontent.com/59583325/195386209-36800350-0fd6-411a-8a76-52088768687d.png)

![Screenshot (172)](https://user-images.githubusercontent.com/59583325/195386330-40858d54-7bfc-4b62-bdda-9f886cfec0f9.png)


## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

vsaleem committed:
[MHV 33641] Added UnAuth MessageFAQ components

-Created reusable UnAuthBanner and UnAuthSendMessageSection for unauthenticated users

vsaleem committed:
[MHV-33641] Created MessageFAQ Container and FAQ sub-component

vsaleem committed:
[MHV-33641] Added MessageFAQ component with unauth render logic

-added logic for unAuthBanner and UnAuthSendMessageSection components, renders only when user is unauthenticated
-updated routes, nav, and breadcrumb for MessageFAQ

vsaleem committed:
[MHV-33641] Updated FAQ component

-uncommented-out FAQ component

vsaleem committed:
Merge branch 'main' into MHV-33641-Messages-FAQ-UNAUTH

oleksii-morgun-va commented on src/applications/mhv/secure-messaging/components/Navigation.jsx:
```diff
@@ -116,6 +116,11 @@ const Navigation = () => {
+                <a href="/my-health/secure-messages/message-faq">
+                  Messages FAQ
+                </a>
+              </li>
```
You may want to consider adding this to `paths` const up above

vsaleem committed:
[MHV-33641] Updated routes

-Added /faq to path const in Navigation.jsx
-Updated routes to /faq in breadcrumb and routes.jsx
